src/Data/toClientParams.ts

```diff
@@ -30,13 +47,55 @@ export function toClientParams(params: IndexParams): FetchParams {
   }
 }
 
-function toClientFilterParam(filters: Record<string, OperatorSpec>) {
+function toClientFilterParam(
+  filters: Record<string, FilterSpec | AndFilterSpec>,
+) {
   const params: Record<string, string> = {}
 
-  Object.entries(filters).forEach(([name, { opCode, value }]) => {
-    const key = opCode === 'eq' ? name : [name, opCode].join('.')
-    params[key] = value
+  Object.entries(filters).forEach(([name, filter]) => {
+    let filterCollection: FilterSpec[]
+
+    if (filter.operator === 'and') {
+      assertNoConflicts(filter.value)
+      filterCollection = filter.value
+    } else {
+      filterCollection = [filter]
+    }
+
+    filterCollection.forEach((currentFilter) => {
```

Comment:
Again, inspired by the SDK.

Reply:
> We're just flattening and operations as if they were top-level.

Is this covered by my [comment](https://github.com/Scrivito/scrivito-portal-app/pull/525#discussion_r1694973663) above? Otherwise please elaborate what you mean by this.

> why don't we receive a flat input already?

Quote from the [concept](https://docs.google.com/document/d/1-BdfNqypQctWiJmaNYHzyHSgboWU7b37nTqrFq__4Qs/edit#heading=h.lchp0odx7lb3):

> But other places model filters as a map from attribute name to operator. In these places, it's not yet possible to have multiple operators on the same attribute.

This applies to the `IndexParams#filters` API. So flattening is not possible here, unless we also change the format to a list (which I cannot derive from the concept)